app/src/main/java/woowacourse/shopping/feature/cart/CartViewModel.kt

```
@@ -146,77 +322,94 @@ class CartViewModel(
-            pageOffset * 10,
-            { response ->
-                val allGoodsList = response.content.map { CartItem(it.toDomain(), 0) }
-
+        viewModelScope.launch {
+            try {
+                val goodsResponse = goodsRepository.fetchPageGoods(10, pageOffset * 10)
+                val allGoodsList = goodsResponse.content.map { CartItem(it.toDomain(), 0) }
                 filterAndSetRecommendedGoods(allGoodsList, pageOffset)
-            },
-            {
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)